spec/services/adjustment_create_service_spec.rb

```diff
@@ -114,11 +125,11 @@
                                "0": {item_id: item_1.id, quantity: quantity}
                              }}
         subject.new(adjustment_params).call
-      end.to change { inventory_item_1.reload.quantity }.by(0)
+      end.to change { inventory_item_1.reload.quantity }.by(0).and change { item1_inventory_quantity }.by(0)
     end
 
     it "gives an error if we attempt to adjust inventory below 0" do
-      quantity = -1 * (inventory_item_1.quantity + 1)
+      quantity = -101
```

Comment:
I like the explicitness of these

Reply:
Me too. The dynamism of our specs really rubs me the wrong way since it's so hard to tell what things are supposed to be. Half the time the specs just copy the code from the thing they're testing. -\_-